ansible/aodn-database/roles/common/library/postgresql_privs2.py

```diff
@@ -0,0 +1,802 @@
+#!/usr/bin/python
```

Comment:
This is a newer version of the postgresql_privs module copied from upstream to support default privileges. Wouldn't need to be here once merged upstream.